src/BenchmarkDotNet.Core/BenchmarkDotNet.Core.csproj

```diff
@@ -8,6 +8,9 @@
     <PackageId>BenchmarkDotNet.Core</PackageId>
     <RootNamespace>BenchmarkDotNet</RootNamespace>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net45|AnyCPU'">
```

Comment:
I didn't mean to change this file, but I'm unable to compile when I revert this change. If you want to, I can revert this file before merging.